packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -27,6 +34,18 @@ export function localizePadding(top: number, right: number, bottom: number, left
     };
 }
 
+export function insetStrongBoxShadow(color: string): ICSSRules<{}> {
```

Comment:
All functions should have comments describing what it is and what is does.

Reply:
Once I understand more about what this does I may suggest a different function name.

Reply:
I actually feel like this would be very redundant. Most of them will simply repeat the title... aka "applyInputStyles" - "Applies the general input styles"... doesn't really add any context.